components/automate-cli/pkg/verifyserver/models/systemuser.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+type SystemUserServiceCheck struct {
+	Title         string `json:"title"`
+	Passed        bool   `json:"passed"`
+	SuccessMsg    string `json:"success_msg"`
+	ErrorMsg      string `json:"error_msg"`
+	ResolutionMsg string `json:"resolution_msg"`
+}
```

Comment:
there is already model named Checks you can reuse it

Reply:
Done